### PR TITLE
new: SimpleOverlay

### PR DESCRIPTION
**Pull request by HosizoraN**
Gameplay:
Screenshot 2024-05-08 16-56-10

ResultScreen:
Screenshot 2024-05-08 16-56-20

CompatibleWith: tosu
Support All Mode! Customizable!

### CODE REVIEW

File: counters/SimpleOverlay by HosizoraN/index.js [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+       if (message['hpColor'] != null) {
+          document.body.style.setProperty('--hpColor', message['hpColor']);
+       };
+
+    } catch (error) {
+      console.log(error);
+    };
+  });
+
+socket.api_v2(({ play }) => {
```
Comment (cyperdark) [inline counters/SimpleOverlay by HosizoraN/index.js:100]: Remove this

File: counters/SimpleOverlay by HosizoraN/index.js [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+          cache.r100 = resultsScreen.hits[100];
+        }
+        if (cache.r50 != resultsScreen.hits[50]) {
+          cache.r50 = resultsScreen.hits[50];
+        }
+        if (cache.r0 != resultsScreen.hits[0]) {
+          cache.r0 = resultsScreen.hits[0];
+        }
+
+        const Folder = folders.beatmap.replace(/#/g, "%23").replace(/%/g, "%25").replace(/\\/g, "/").replace(/'/g, "%27").replace(/ /g, "%20");
```
Comment (cyperdark) [inline counters/SimpleOverlay by HosizoraN/index.js:403]: you can use [encodeURI](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/encodeURI)

idk if it's working, but worth trying

File: counters/SimpleOverlay by HosizoraN/settings.values.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"ModeID":"Mania","KeyEnable":true,"HideUI":false,"KeyColor":"#ffffff","keyPressedColor":"#ffd256","TextColor":"#ffffff","keyTextColor":"#ffffff","katuColor":"#ffffff","h100Color":"#ffffff","h50Color":"#ffffff","h0Color":"#ffffff","hSBColor":"#ffffff","FontID":"Montserrat","hpColor":"#ffffff"}
```
Comment (cyperdark) [inline counters/SimpleOverlay by HosizoraN/settings.values.json:1]: remove this file

File: counters/SimpleOverlay by HosizoraN/index.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                    <div id="rcombo"></div>
+                </div>
+                <div id="rank"></div>
+                <div id="modsused"></div>
+            </div>
+        </div>
+    </div>
+
+    <!--        DO NOT REMOVE THOSE        -->
+    <script src="js/reconnecting-websocket.min.js"></script>
```
Comment (cyperdark) [inline counters/SimpleOverlay by HosizoraN/index.html:78]: not used anywhere

**Review by cyperdark — CHANGES_REQUESTED** [review]

File: counters/SimpleOverlay by HosizoraN/index.js (hunk 2 above)
Comment (HosizoraN) [reply counters/SimpleOverlay by HosizoraN/index.js:403]: i did try that and only space (%20) doesn't removed so it didn't worked out

image

**Pull request merged**